lib/adaptations/WordpressSelfSigned.ts

```diff
@@ -31,12 +31,6 @@ export class SelfSignedWordpressEcsConstruct extends WordpressEcsConstruct {
       loadBalancerName: undefined,
       loadBalancer: alb,
    });
-
-    // The apache service running in the wordpress container needs to have a virtual host server name 
-    // that matches the alb public address
-    if(this.containerDefProps?.environment) {
```

Comment:
If this comes out, does that mean that a deploy without cloud formation will no longer get a virtual host server name that matches the alb public address?

Reply:
No, I simply moved the assignment of SERVER_NAME as a container environment variable out into Wordpress.ts